setuptools/_vendor/vendored.txt

```diff
@@ -2,3 +2,5 @@ packaging==21.2
 pyparsing==2.2.1
 ordered-set==3.1.1
 more_itertools==8.8.0
+tomli==1.2.3
+# validate-pyproject[all]==0.3.2  # Special handling, don't remove
```

Comment:
Is this meant to be commented out?

Reply:
Yes, there is some code in `tools/vendoring` that will handle this entry in a special way.

`validate-pyproject` itself implements some vendoring logic that minimises the amount to external dependencies.

Reply:
Do you have any suggestion about how to write the comment in a way to make this more clear?

Reply:
Maybe say exactly that, "Handled by tools/vendoring"

Reply:
I gave it a try, please let me know if that works :)